README.md

```diff
@@ -143,13 +143,21 @@ def hello(data, context):
     print(context)
 ```
 
-To enable automatic unmarshalling, set the function signature type to `event` using the `--signature-type` command-line flag or the `FUNCTION_SIGNATURE_TYPE` environment variable. By default, the HTTP signature type will be used and automatic event unmarshalling will be disabled.
+The Functions framework can also unmarshall incoming [CloudEvents](http://cloudevents.io) payloads to the `cloudevent` object. This will be passed as a [cloudevent](https://github.com/cloudevents/sdk-python) to your function when it receives a request.  Note that your function must use the cloudevents-style function signature:
+
+```python
+def hello(cloudevent):
+    print(f"Received event with ID: {cloudevent['id']}")
+```
+
+To enable automatic unmarshalling, set the function signature type to either `event` or `cloudevent` using the `--signature-type` command-line flag or the `FUNCTION_SIGNATURE_TYPE` environment variable. By default, the HTTP signature type will be used and automatic event unmarshalling will be disabled.
```

Comment:
Only the `cloudevent` signature should support CloudEvents. The `events` signature should not support CloudEvents.

Reply:
Ah I see. Removed!